### ISSUE TITLE
Latest Update did not fix File Size issues

### ISSUE THREAD

**Issue by f-guichard** [kwsch/PKHeX#4019]
**Describe the bug**
The latest update which said that the file size issue for Violet's latest update has been fixed, has not been fixed.

**To Reproduce**

1. Open save file
2. Get hit with the following:
![pkmon_violet_2 0 1_err_input_file_2_large](https://github.com/kwsch/PKHeX/assets/14328145/24dcc155-ef1e-48a1-8dc4-c1b1d2118d47)

**Expected behavior**
For the save game to be loaded.

**Screenshots**
See above.

**Additional context**
None.

**Comment by LokeYourC3PH** [comment]
Bro, did you copy all my text 😂?
Anyways, it's fixed in https://github.com/kwsch/PKHeX/commit/0cda774b97c541373872ab804e6c6d3cb7f7af52

**Comment by kwsch** [comment]
That filesize (0x436C22, 4418594) is way outside the [range of valid save files](https://github.com/kwsch/PKHeX/blob/0cda774b97c541373872ab804e6c6d3cb7f7af52/PKHeX.Core/Saves/Util/SaveUtil.cs#L39-L64).

I don't believe you've correctly exported your save file from your console.

**Comment by LokeYourC3PH** [comment]
> That filesize (0x436C22, 4418594) is way outside the [range of valid save files](https://github.com/kwsch/PKHeX/blob/0cda774b97c541373872ab804e6c6d3cb7f7af52/PKHeX.Core/Saves/Util/SaveUtil.cs#L39-L64).
> 
> I don't believe you've correctly exported your save file from your console.

LMAO YES, I see it too now, holy hell it's huuuuuuuge.
@f-guichard You might want to use JKSV again and export your save once more, see if it still continues that way.

**Comment by f-guichard** [comment]
> Bro, did you copy all my text 😂?
Yes, i went through your own issue, and i was dealing with the exact same trouble, i decided to reuse your issue like a template, hope i was not mistaken.
> That filesize (0x436C22, 4418594) is way outside the range of valid save files
I cloned the repo and went through SaveUtil.cs, i basically thought you had to add hex-encoded string of the filesize.
You are the code base owner, so you know your algorithms & functions better than anybody else !
I made this export process 2 times, but i will do it at least 5 times to have a "statistically" more robust feedback. 
> You might want to use JKSV again and export your save once more, see if it still continues that way.
@LokeYourC3PH Thx for the suggestion, i already did that multiple times, last time last evening :)

@LokeYourC3PH @kwsch Thank you for your remarks and feedbacks, i hope it will be, as kwsch suggested, a trouble with save files exports. 
I will provide this feedback tonight.
